hoomd/hpmc/UpdaterMuVT.h

```diff
@@ -643,15 +644,29 @@ void UpdaterMuVT<Shape>::update(unsigned int timestep)
                     Scalar3 f;
                     f.x = rng.template s<Scalar>();
                     f.y = rng.template s<Scalar>();
-                    f.z = rng.template s<Scalar>();
+                    if (m_sysdef->getNDimensions() == 2)
+                        {
+                        f.z = Scalar(0.5);
+                        }
+                    else
+                        {
+                        f.z = rng.template s<Scalar>();
+                        }
                     vec3<Scalar> pos_test = vec3<Scalar>(m_pdata->getGlobalBox().makeCoordinates(f));
 
                     Shape shape_test(quat<Scalar>(), param);
                     if (shape_test.hasOrientation())
                         {
                         // set particle orientation
+                        if (m_sysdef->getNDimensions() == 2)
+                        {
+                        shape_test.orientation = generateRandomOrientation2D(rng);
+                        }
+                        else
+                        {
                         shape_test.orientation = generateRandomOrientation(rng);
                         }
```

Comment:
Indent the bodies of the conditional blocks.

Reply:
👍